model/config.go

```diff
@@ -204,6 +204,7 @@ type ServiceSettings struct {
 	EnableChannelViewedMessages              *bool
 	EnableUserStatuses                       *bool
 	ClusterLogTimeoutMilliseconds            *int
+	CloseUnusedDirectMessagesInSidebar       *bool
```

Comment:
0/5 here, but maybe we can remove `InSidebar` as that only refers to the webapp

Reply:
Completely agree. The JIRA ticket was just very specific about the name (It wasn't my choosing.).

@esethna I assume it's okay if we deviate from that a bit?

Reply:
Yes that's fine @ccbrown, we can go with "CloseUnusedDirectMessages"